doc/source/general/gui.rst

```diff
@@ -1,4 +1,4 @@
-.. _gui:
+.. _gui_overview:
```

Comment:
It took a lot of trial and error to figure this one out:  It gets very confused if you include a label tag of ``_foo`` inside your ``foo.rst`` file when you also have a folder alongside it called ``foo/`` with other files in it.

When you tell Sphinx to automatically build a toctree from a folder, it also makes that folder name into a label tag.  Having both the two tags caused it to double everything in the folder walk, and mention it twice in the table of contents on the left, and mention it at the wrong nesting level.  It took me a while to work out that I had to rename this tag to fix that.  (Sphinx never issued any complaint about it.  It was happy to have the same file in two different places of the tree, so I had a hard time figuring out the cause.)